### PR TITLE
Show bad configured CI

### PR DESCRIPTION
**Pull request by kolibri91**
@manuzhang 

Your CI is green even if it contains styling-issues:
e.g. https://github.com/manuzhang/mkdocs-htmlproofer-plugin/actions/runs/4637384298/jobs/8206224207?pr=55
image

There are two issues:
* You should specify folder names instead of `.`
* `--exit-zero` is most likely not want you want

I'm going to solve all issues with my next PR. I noticed it after my implementation, so it would be easier to adress it there.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,9 +28,9 @@ jobs:
       - name: Lint with flake8
         run: |
           # stop the build if there are Python syntax errors or undefined names
-          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          flake8 htmlproofer/ tests/ setup.py --count --select=E9,F63,F7,F82 --show-source --statistics
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+          flake8 htmlproofer/ tests/ setup.py --count --max-complexity=10 --max-line-length=127 --statistics
```
Comment (manuzhang) [inline .github/workflows/ci.yml:33]: the comment above needs update as well

**Comment by manuzhang** [comment]
Please help fix style issues after removing `--exit-zero`

**Comment by kolibri91** [comment]
> Please help fix style issues after removing `--exit-zero`

Really? It's done in #56. So, why to double the work?

**Comment by manuzhang** [comment]
I prefer one PR/commit handling one issue. Combing multiple things in one commits make it hard to check history, revert, etc. You may have two commits in #57 with one handling style, the other external URL.

**Comment by kolibri91** [comment]
Fair point, I would do it in the same way if it's really matters. But the project is rather small and the history is easy to overview.

Nevertheless, I will rework the commits

**Comment by kolibri91** [comment]
Styling issues are fixed by 5b230b8217c220af535ef328ac6a4457ab11ed82 & ed1dd7f7abd5164c0d0ccb6f50c743ac90f4277c

**Review by manuzhang — APPROVED** [review]

**Pull request merged**